packages/vuetify/src/components/VDataTable/VDataTableHeaderDesktop.ts

```diff
@@ -16,76 +16,95 @@ export default mixins(header).extend({
         },
       }, ['group'])
     },
-    // eslint-disable-next-line max-statements
+    getAria (beingSorted: boolean, isDesc: boolean) {
+      let ariaSort = 'none'
+      let ariaLabel = [
+        this.$vuetify.lang.t('$vuetify.dataTable.ariaLabel.sortNone'),
+        this.$vuetify.lang.t('$vuetify.dataTable.ariaLabel.activateAscending'),
+      ]
+
+      if (beingSorted) {
```

Comment:
Assigning a default object:

```ts
const aria = { ariaSort: 'none', ariaLabel: '' }
```

Means that we can return early and avoid the nested conditionals:

```ts
if (!beingSorted) {
  aria.ariaLabel = ariaLabel.join(' ')
  
  return
}
```